samples/ComponentExamples/eslint.config.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  baseConfig,
+  baseGlobalIgnores,
+  globalConfig,
+  testFilesBaseConfig
+} from '../../common/config/ESLint/eslint.config';
+import { globalIgnores } from 'eslint/config';
+
+export default globalConfig([
+  baseConfig,
+  testFilesBaseConfig,
+  {
+    rules: {
+      'headers/header-format': 'off',
+      'jsdoc/require-jsdoc': 'off'
+    }
+  },
+  {
+    linterOptions: {
+      reportUnusedDisableDirectives: false
```

Comment:
Perhaps turning back on the rules that were enabled will enable this line to be left out?

Reply:
I was able to validate I didnt need this block so I am removing it.